lib/AppInfo/Application.php

```diff
@@ -46,6 +46,7 @@
 use OCP\IServerContainer;
 use Throwable;
 
+require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Do we really want to move from the default composer/ if we need to add this line ?

Reply:
I will merge this one, so we stop going back and forth with this. While waiting for @ChristophWurst point of view on the subject

Reply:
This is fine IMO. I would not say that Nextcloud is responsible for registering a possible custom autoloader of the app, so this is the most appropriate location to do this.